signac_dashboard/modules/navigator.py

```diff
@@ -0,0 +1,99 @@
+from flask import render_template, url_for
+
+from signac_dashboard.module import Module
+from signac_dashboard.util import escape_truncated_values
+
+
+class Navigator(Module):
+    """Displays links to jobs differing in one state point parameter.
+
+    This module may not update if the signac project changes because the module
+    caches the project schema when the signac-dashboard launches.
+
+    :param context: Supports :code:`'JobContext`
+    :type context: str
+    """
+
+    _supported_contexts = {"JobContext"}
+
+    def __init__(
+        self,
+        name="Navigator",
+        context="JobContext",
+        template="cards/navigator.html",
+        **kwargs,
+    ):
+        super().__init__(name=name, context=context, template=template, **kwargs)
+
+    def get_cards(self, job):
+        project = self._dashboard.project
+
+        def link_label(key, other_val):
+            """Returns the url and label for the job with job.sp[key] == other_val."""
+
+            similar_sp = job.sp()  # modifiable
```

Comment:
I have a weak preference for using the full names like `statepoint` instead of `sp` in signac package internals.
```suggestion
            similar_statepoint = job.statepoint()  # modifiable
```